src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,27 @@
+import { User } from '../../types/User';
+
+type Props = {
+  user: User,
+  completed: boolean,
+};
+
+export const UserInfo: React.FC<Props> = ({ user, completed }) => {
+  const {
+    name,
+    email,
+  } = user;
+
+  return (
+    <p className="block" data-cy="modal-user">
+      {completed
+        ? <strong className="has-text-success">Done</strong>
+        : <strong className="has-text-danger">Planned</strong>}
```

Comment:
Use ternary to display only text inside `strong`. Use classNames to set proper style.